src/lib/features/event-page/EventPage.svelte

```diff
@@ -0,0 +1,95 @@
+<script type="ts">
+	import Blur from '$lib/components/Blur.svelte';
+	import FloatTicket from '$lib/components/floats/FloatTicket.svelte';
+	import type { Event } from '$lib/types/event/event.interface';
+	import { Button } from '@emerald-dao/component-library';
+
+	export let event: Event;
+</script>
+
+<section class="container">
+	<div class="main-wrapper">
+		<div class="side-wrapper">
+			<h4>{`#${event.eventId}`}</h4>
+			<p class="small">Event ID</p>
+		</div>
+		<div class="event-wrapper">
+			<Blur color="tertiary" right="15%" top="10%" />
+			<Blur left="15%" top="10%" />
+			<FloatTicket float={event} />
+		</div>
+		<div class="side-wrapper">
+			<h4>{`${event.totalSupply}`}</h4>
+			<p class="small">FLOATs claimed</p>
+		</div>
+	</div>
+	<div class="container-small details-wrapper">
+		<div>
+			<p class="large">{`${event.dateCreated}`}</p>
+			<p class="small">Start Date</p>
+		</div>
+		<div>
+			<p class="large">06/30/2023</p>
```

Comment:
@IgnacioDebat any reason why this is hardcoded??

Reply:
I found the reason chino!

There is no timeframe property on the Event interface.
That's because the timeframe is an optional verifier that goes inside the array of verifiers (see Event interface)

Maybe we can add some conditions so that this component reads if there is a timeframe verifier and if there is one use those dates!